cmd/list.go

```diff
@@ -0,0 +1,30 @@
+package cmd
+
+import 	(
+	"fmt"
+	"os"
+	"github.com/spf13/cobra"
```

Comment:
Put an empty line between these lines to separate the stdlib imports from external imports

Reply:
This still needs to be done

Reply:
woops fixed it